api/v1alpha1/physical_machine_chaos_types.go

```diff
@@ -0,0 +1,298 @@
+// Copyright 2021 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// PhysicalMachineChaosAction represents the chaos action about physical machine.
+type PhysicalMachineChaosAction string
+
+const (
+	// PM means physical machine
+	PMStressCPUAction PhysicalMachineChaosAction = "stress-cpu"
```

Comment:
I think these const vars may not be used in this pr, maybe add them into doc or comment?

Reply:
delete in https://github.com/chaos-mesh/chaos-mesh/pull/2235/commits/1586aabe3a0034a5a8c295bf15bd02983dbefce6